docs/ref/models/expressions.txt

```diff
@@ -228,6 +228,22 @@ primary key value rather than a model instance::
     >> car.built_by
     3
 
+.. _using-f-to-sort-nulls-last:
+
+Using ``F()`` to sort nulls last
```

Comment:
I'd chop "last" from the name and perhaps phrase it as "... to sort null values".